cmp/chart/Chart.js

```diff
@@ -251,7 +251,11 @@ class LocalModel extends HoistModel {
             enabled: false,
             fallbackToExportServer: false,
             chartOptions: {
-                scrollbar: {enabled: false}
+                scrollbar: {enabled: false},
+                rangeSelector: {enabled: false},
+                navigator: {enabled: false},
+                xAxis: [{labels: {enabled: true}}],
```

Comment:
Certainly some charts may have more than one x or y axis, but those are pretty rare, and will require quite a bit of custom config, so custom export config can be expected on those too.

This seemed like a sane default for the most frequent use case, without adding array merge complexity.
Same comment for print export below.